theano/gof/vm.py

```diff
@@ -809,13 +820,18 @@ def make_vm(self, nodes, thunks,
         pre_call_clear = [storage_map[v] for v in self.no_recycling]
 
         if (self.callback is not None or
-                (config.profile and config.profile_memory)):
+                (config.profile and config.profile_memory) or
+                self.allow_partial_eval):
```

Comment:
do getattr(self, 'allow_partial_eval', False) to work with old pickled VM_Linker instance. This make travis fail.